packages/terra-form-input/tests/nightwatch/InputTestRoutes.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { Route } from 'react-router';
+import InputTests from './InputTests';
+
+// Test Cases
+import {
+  DefaultInput,
+  InvalidInput,
+  DisabledInput,
+} from './InputPropPermutations';
+
+const routes = (
+  <div>
```

Comment:
We need to add links to test for `onChange`, `onBlue`, `onFocus`, and `required` props

Reply:
Done c33141d942293cae899fc6e6efcd0587346cc7db